### ISSUE TITLE
Generating vocabulary only from the training set

### ISSUE THREAD

**Issue by sxs4337** [tsenghungchen/SA-tensorflow#5]
The vocabulary should be generated only using the training data.
Currently, in function-
https://github.com/tsenghungchen/SA-tensorflow/blob/master/Att.py#L370 , the input is "captions" which is generated from all data- train+val+test.
Ideally, the network should not be fed any words from the test set (any unseen new words in testing to the network should be just <unknown_word> for evaluation).
Thanks.

**Comment by tsenghungchen** [comment]
Yeah, you're right. Thank you for pointing it out. I'll update the code.
